lib/analysis_options.1.0.0.yaml

```diff
@@ -11,12 +11,14 @@ linter:
     - avoid_function_literals_in_foreach_calls
     - avoid_init_to_null
     - avoid_null_checks_in_equality_operators
+    - avoid_print
```

Comment:
can we create `analysis_options.1.0.1.yaml` with these rules to maintain backward compatibility?

Reply:
Changes LGMT, but I agree with @felangel that it should be versioned ;)

Reply:
Updated